.gitlab-ci.yml

```diff
@@ -721,24 +721,24 @@ library:ci-mczify:
   - library:ci-mathcomp
 
 library:ci-mathcomp_base:
-  extends: .ci-template
+  extends: .ci-template-flambda
```

Comment:
The point of mathcomp_base is that it's not flambda, otherwise there's already the mathcomp job

Reply:
Another point could so that things that only depend on mathcomp_base can be run faster. Could we make `mathcomp` depend on `mathcomp_base` to avoid rebuilding the same thing twice?

Reply:
Oh I though the point was to do a partial build, as the math-comp base is pretty larguish. But indeed let's remove the duplication.